backend/src/MessageHandler/CheckAndFetchAllFeedsHandler.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace App\MessageHandler;
+
+use App\Entity\Publication;
+use App\Entity\PublicationFetch;
+use App\Enum\FetchStatusEnum;
+use App\Message\CheckAndFetchAllFeedsMessage;
+use App\Repository\PublicationRepository;
+use App\Service\FeedItemProcessor;
+use App\Service\Parser\Parser;
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Messenger\Attribute\AsMessageHandler;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+#[AsMessageHandler]
+class CheckAndFetchAllFeedsHandler
+{
+    public function __construct(
+        private PublicationRepository $publicationRepository,
+        private EntityManagerInterface $entityManager,
+        private FeedItemProcessor $itemProcessor,
+        private HttpClientInterface $httpClient,
+        private LoggerInterface $logger,
+    ) {
+    }
+
+    public function __invoke(CheckAndFetchAllFeedsMessage $message): void
+    {
+        $this->logger->info('Starting scheduled feed check');
+        
+        $duePublications = $this->publicationRepository->findDueForFetching(new \DateTime());
+        
+        if (empty($duePublications)) {
+            $this->logger->info('No publications due for fetching');
+            return;
+        }
+
+        $this->logger->info('Found publications due for fetching', ['count' => count($duePublications)]);
+        
+        foreach ($duePublications as $publication) {
+            $this->processFeed($publication);
```

Comment:
here, we should dispatch a message for each publication, and update the nextFetchAt based on interval.